galette/lib/Galette/Features/Replacements.php

```diff
@@ -226,6 +226,7 @@ protected function getMainPatterns(): array
                 'pattern'          => '/{ASSO_LOGO}/',
             ],
             'date_now'              => [
+                //TRANS: see https://www.php.net/manual/datetime.format.php
```

Comment:
Comments will be displayed in weblate UI
![image](https://user-images.githubusercontent.com/224733/221930629-9847bb9b-ddd3-4d6a-8fa8-a65aa49e7ba0.png)

Reply:
I really should have a look at weblate one day ! :sweat_smile: